pkg/rpm_pfg.bzl

```diff
@@ -659,7 +659,17 @@ def _pkg_rpm_impl(ctx):
         tools = tools,
     )
 
+    changes = []
+    if ctx.attr.changelog:
+        changes = [ctx.attr.changelog]
+
+    output_groups = {
+        "out": [default_file],
+        "rpm": [output_file],
+        "changes": changes
+    }
     return [
+        OutputGroupInfo(**output_groups),
         DefaultInfo(
             files = depset(outputs),
```

Comment:
The right change here is to align with the other tools.  It should be
```
DefaultInfo(files = depset([output_file]), runfiles = ctx.runfiles(files = outputs))
```

BUT... that breaks existing tests.  So, what don't we reduce this CL to just the changes to this file, which are reasonable, and come back to the problem of pkg_rpm giving two output files later.